js/src/forum/addComposerAutocomplete.js

```diff
@@ -11,6 +12,16 @@ export default function addComposerAutocomplete() {
 
   const emojiKeys = Object.keys(emojiMap);
 
+  function getEmojiIconCode(emoji) {
```

Comment:
using only `toCodePoint()` breaks some emojis and twemoji does not export [this function](https://github.com/twitter/twemoji/blob/0dd7c46f41837f1aff7ae930e1993e20d66ddd62/2/twemoji.npm.js#L322) so I had to recreate it.

should we use something like this for emojimap instead:

```js
[
	{
		"emoji": "😂 ",
		"code": "1f602",
		"aliases": ["alias_1", "alias_2", "alias_3"],
	}
	// ...
]
```

Reply:
If recreating the twemoji function is reliable, then no need to pollute the emoji map with extra information?

Reply:
I think it's working fine